tck/src/test/resources/Token1.json

```diff
@@ -3,8 +3,8 @@
     "jti": "a-123",
     "sub": "24400320",
     "upn": "jdoe@example.com",
-    "preferred_username": "jdoe",
-    "aud": "s6BhdRkqt3",
+    "preferred_username": "jdoe",    
```

Comment:
Extra space in the end of `preferred_username` and just before `custom_string`

Reply:
fixed